config/v1/types_feature.go

```diff
@@ -200,6 +200,7 @@ var defaultFeatures = &FeatureGateEnabledDisabled{
 	},
 	Disabled: []FeatureGateDescription{
 		retroactiveDefaultStorageClass,
+		validatingAdmissionPolicy,
```

Comment:
this shouldn't be specified. don't want to keep it disabled once it is enabled by default.  You can specify it as disabled in techpreview, which is weird, but I think will do the same thing.  test in a fake PR to cluster-config-operator.

Reply:
Thanks, done and it verified that it shows up in the disabled list for Default as expected.